### ISSUE TITLE
Throw `ArgumentError.notNull` instead

### ISSUE THREAD

**Issue by kevmoo** [google/built_value.dart#42]
Code generated now...

``` dart
  _$GitHubRepoData._({this.org, this.repo}) : super._() {
    if (org == null) throw new ArgumentError('null org');
    if (repo == null) throw new ArgumentError('null repo');
  }
```

use `new ArgumentError.notNull` – is nice

**Comment by davidmorgan** [comment]
https://github.com/google/built_value.dart/pull/47

**Comment by kevmoo** [comment]
Point for using "Fixes #42" in the body of the commit. GitHub will close it for you. 😄
